tests/test_EEE_module/test_emissions.py

```diff
@@ -0,0 +1,426 @@
+                "harvest_year",
+                "harvest_day",
+            ),
+            mocker.call(
+                {
+                    "name": "Fertilizer Applications",
+                    "description": "Collects all synthetic fertilizer applications that occurred in the simulation.",
+                    "filters": ["Field._record_fertilizer_application\\.fertilizer_application\\.field='.*'"],
+                    "variables": [".*"],
+                    "date_fields": ("year", "day"),
+                },
+                datetime(2014, 1, 20),
+                "year",
+                "day",
+            ),
+            mocker.call(
+                {
+                    "name": "Manure Applications",
+                    "description": "Collects all manure applications that occurred in the simulation.",
+                    "filters": ["Field._record_manure_application\\.manure_application\\.field='.*'"],
+                    "variables": [".*"],
+                    "date_fields": ("year", "day"),
+                },
+                datetime(2014, 1, 20),
+                "year",
+                "day",
+            ),
+            mocker.call(
+                {
+                    "name": "Manure Requests",
+                    "description": "Collects all manure requests that occurred in the simulation.",
+                    "filters": ["Field._record_manure_application\\.manure_request\\.field='.*'"],
+                    "variables": [".*"],
+                    "date_fields": ("year", "day"),
+                },
+                datetime(2014, 1, 20),
+                "year",
+                "day",
+            ),
+        ]
+    )
+
+
+def test_gather_ration_feed_totals(mocker: MockerFixture) -> None:
+    """
+    Test that the totals of feeds from rations given to animals in the last 365 days of the simulation are correct.
+    """
+    em = EmissionsEstimator()
+    mock_filter = mocker.patch.object(em.om, "filter_variables_pool", return_value={"test": {"values": [2, 3, 4]}})
+    expected = {"test": 9.0}
+    observed = em._gather_ration_feed_totals()
+    assert observed == expected
+    mock_filter.assert_called_once_with(
+        {
+            "name": "Feed Ration Totals",
+            "description": "Gathers the amounts of purchased feeds fed to animals in the last year of the simulation.",
+            "filters": ["AnimalModuleReporter.report_daily_ration.ration_daily_feed_totals.*"],
+            "variables": [r"^\d+$"],
+            "slice_start": -365,
+        }
+    )
+
+
+def test_transform_outputs_to_list_of_dicts() -> None:
+    """Test that the function transform data to correct list of dicts."""
+    em = EmissionsEstimator()
+    expected = [{"one": 1, "two": 4}, {"one": 2, "two": 5}, {"one": 3, "two": 6}]
+    data = {"one": {"values": [1, 2, 3]}, "two": {"values": [4, 5, 6]}}
+    observed = em._transform_outputs_to_list_of_dicts(data)
+    assert observed == expected
```

Comment:
```suggestion
    expected = [{"one": 1, "two": 4}, {"one": 2, "two": 5}, {"one": 3, "two": 6}]
    data = {"one": {"values": [1, 2, 3]}, "two": {"values": [4, 5, 6]}}
    
    observed = em._transform_outputs_to_list_of_dicts(data)
    
    assert observed == expected
```
I also suggest making "data" and "expected" parameters and testing more cases. What if the data passed is empty? What if it has info maps in it? Possibly combining this test with the one below it and making "expect_error" a parameter as well.

Reply:
I do not understand the info maps part. Can you explain a bit more?

Reply:
When data is pulled out of the of the Output Manager, it may or may not have info maps in it. I.e. you might see 
`{"one": {"values": [1, 2, 3]}, "two": {"values": [4, 5, 6]}}` or `{"one": {"values": [1, 2, 3], "info_maps": [{...}, {...}, {...}]}, "two": {"values": [4, 5, 6], "info_maps": [{...}, {...}, {...}]}}`